baseimages/docker.Windows.Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Required.
+# docker build -f baseimages/docker.Windows.Dockerfile -t docker .
+FROM microsoft/windowsservercore as download
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV DOCKER_VERSION 18.03.0-ce
+
+RUN Invoke-WebRequest 'http://constexpr.org/innoextract/files/innoextract-1.6-windows.zip' -OutFile 'innoextract.zip' -UseBasicParsing ; \
+    Expand-Archive innoextract.zip -DestinationPath C:\ ; \
+    Remove-Item -Path innoextract.zip
+
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; \
```

Comment:
>[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12  [](start = 4, length = 80)

you can combine all related RUN commands so that number of layers gets reduced as RUN, COPY, ADD create layers.

Reply:
It makes sense to me to leave this as-is, since you'll break the cache of the zip utility if you change the toolbox release version with that approach.